.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+  "editor.formatOnSave": true,
+  "editor.tabCompletion": true
```

Comment:
we can also disable some things that people may have set up in their own env. E.g. if someone uses prettier for formatting, we wouldn't want that to conflict with what is listed here, so we can set prettier to false.

Reply:
Is this one supposed to be true/false? On my local, it says it has to be a string `"on"` or `"off"`

Suggest we also add:

`  "files.insertFinalNewline": true`